port/osx/omrosbacktrace_impl.c

```diff
@@ -341,6 +344,11 @@ omrintrospect_backtrace_thread_raw(OMRPortLibrary *portLibrary, J9PlatformThread
 	}
 
 	return i - discard;
+#elif defined(OMR_ARCH_AARCH64) /* defined(OMR_ARCH_X86) */
+	return 0;
+#else
```

Comment:
https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#comment-conditional-compilation-directives-if-ifdef-ifndef-else-elif-endif
```suggestion
#else /* defined(OMR_ARCH_AARCH64) */
```